tests/unit/test_quantity.py

```diff
@@ -233,7 +233,9 @@ def test_gt():
     assert jnp.array_equal(q > Quantity(1, u.m), jnp.array([False, True, True]))
 
     # Test with incompatible units
-    assert jnp.array_equal(q > Quantity(0, u.s), jnp.array([False, False, False]))
+    # TODO: better equinox exception matching
```

Comment:
Here and below: Also add tests that `q > 0`, `q >= 0`, etc. should work even without units?

Reply:
SGTM.
Is this fix alight except for adding some tests? It changes exception types.

Reply:
@adrn, resolved?